### DISCUSSION TITLE
Can you use a selector to select an existing note for an input field?

### DISCUSSION THREAD

**Discussion by TheHuntyBadger** [chhoumann/quickadd#302] [Q&A] [answered]
I created a workflow to pull in articles and posts from online and am using Annotator to highlight the PDFs files.

I 'capture' the article or post by printing to PDF in FireFox. This allows me to cut out the ads that can be present in through out an article. I then use Annotator to mark it up as  Reference and Bibliographic notes for a Zettelkasten system. 

To use Annotator, the front matter needs a value to point to the PDF you want to annotate and has to be manually entered. This can be an online PDF or one in your vault.

I was thinking I could create a template to create a new Annotator Note by having a QuickAdd macro action to open the Template and ask for the PDF Note to annotate. And then populate the front matter value with it. The last step in the Macro would be to open the new note.

**Comment by chhoumann** [comment, marked as answer]
Hi @TheHuntyBadger 

You can create a Template with the following:
``````markdown
---
```js quickadd
const pdfFiles = app.vault.getAllLoadedFiles().filter(f => f.path.endsWith('.pdf'));
const pdfFile = await this.quickAddApi.suggester(pdfFiles.map(f => f.basename), pdfFiles);

return `annotation-target: ${pdfFile.path}`;
```
---

The rest of your template goes here...
``````

This will ask you, from all the PDF files in your vault, which one to choose. Then it'll put the path to it next to `annotation-target` in your frontmatter.

And if you use a template, you can have that open it for you.

**Reply by TheHuntyBadger** [reply]
Thank you so much!
